.env

```diff
@@ -1,4 +1,5 @@
 COMPOSE_PROJECT_NAME=data-science-dags
+IMAGE_REPO=elifesciences/data-science-dags
```

Comment:
Didn't we already do that when pushing the image? Maybe something needs to be merged from develop or so?

Reply:
Ah yes!